pkg/collector/corechecks/ebpf/oom_kill.go

```diff
@@ -106,12 +106,11 @@ func (m *OOMKillCheck) Run() error {
 
 	triggerType := ""
 	triggerTypeText := ""
-	for _, lineRaw := range data {
-		line, ok := lineRaw.(oomkill.Stats)
-		if !ok {
-			log.Error("Raw data has incorrect type")
-			continue
-		}
+	oomkillStats, ok := data.([]oomkill.Stats)
+	if !ok {
+		return log.Errorf("Raw data has incorrect type")
+	}
+	for _, line := range oomkillStats {
```

Comment:
should we create a separated PR for this change for traceability ?

Reply:
This change in the `oom_kill` check has been made necessary by by the signature change of
```go
func (r *RemoteSysProbeUtil) GetCheck(check string) (interface{}, error)
```
in https://github.com/DataDog/datadog-agent/pull/6625/files#diff-f243d7dbf7db8c65cfb24f438bcd9b8a7133b8d93663565c77d66cf6723caadeR20

It now returns an `interface{}` instead of a `[]interface{}` because the `tcp_queue_check` now returns a map instead of a list.